### PR TITLE
Release/v0.9.0

### PR DESCRIPTION
**Pull request by mkondratek**

### CODE REVIEW

**Comment by PawelLipski** [comment]
Pls update CHANGE-NOTES ;)

File: uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala [hunk 1]
```diff
@@ -2,18 +2,40 @@ package com.virtuslab.gitmachete.uitest
...
+  @AfterClass override final def teardown(): Unit = {
+    try {
+      super.teardown()
+    } catch {
+      case e: Exception =>
+        val filteredSuppressed = e.getSuppressed.filterNot { s =>
+          // A duct tape to ignore https://github.com/VirtusLab/ide-probe/issues/95
+          s.getMessage.contains("scala.MatchError: IDE_UPDATE (of class com.intellij.notification.NotificationType)") ||
+            // Another duct tape to ignore a spurious error in the IDEA itself (probably some race condition)
+            s.getMessage.contains("com.intellij.diagnostic.PluginException: Cannot create class com.intellij.uast.UastMetaLanguage")
```
Comment (PawelLipski) [inline uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala:28]: Lemme add one more ignored error here...

File: CHANGE-NOTES.html [hunk 2]
```diff
@@ -1,28 +1,20 @@
-<h3>v0.8.1</h3>
+<h3>v0.9.0</h3>
```
Comment (mkondratek) [inline CHANGE-NOTES.html:1]: @PawelLipski  anything else here?

Comment (PawelLipski) [reply CHANGE-NOTES.html:1]: Nit: there are two sections now (unnamed and `Bug fixes/improvements`)... I'd split it to `Bug fixes` and `Improvements`

Comment (PawelLipski) [reply CHANGE-NOTES.html:1]: Plus `Added support for IntelliJ 2021.1`.

Comment (PawelLipski) [reply CHANGE-NOTES.html:1]: Not mentioning support for Android Studio yet coz I'm not sure if it's gonna work. I also haven't updated `It is compatible with all JetBrains products except Android Studio` in README.md for the same reason

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**